docs/contributing/contribution-checklists.md

```diff
@@ -488,6 +488,180 @@ guidelines.
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheck{Service}TagDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAcc{Service}TagConfig(rName, "key1", "value1"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck{Service}TagExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "key", "key1"),
+					resource.TestCheckResourceAttr(resourceName, "value", "value1"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccAWS{Service}Tag_disappears(t *testing.T) {
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "aws_{service}_tag.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheck{Service}TagDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAcc{Service}TagConfig(rName, "key1", "value1"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck{Service}TagExists(resourceName),
+					testAccCheckResourceDisappears(testAccProvider, resourceAws{Service}Tag(), resourceName),
+				),
+				ExpectNonEmptyPlan: true,
+			},
+		},
+	})
+}
+
+func TestAccAWS{Service}Tag_Value(t *testing.T) {
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "aws_{service}_tag.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheck{Service}TagDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAcc{Service}TagConfig(rName, "key1", "value1"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck{Service}TagExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "key", "key1"),
+					resource.TestCheckResourceAttr(resourceName, "value", "value1"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAcc{Service}TagConfig(rName, "key1", "value1updated"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck{Service}TagExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "key", "key1"),
+					resource.TestCheckResourceAttr(resourceName, "value", "value1updated"),
+				),
+			},
+		},
+	})
+}
+
+func testAcc{Service}TagConfig(rName string, key string, value string) string {
+	return fmt.Sprintf(`
+resource "aws_{service}_{thing}" "test" {
+  name = %[1]q
+
+  lifecycle {
+    ignore_changes = [tags]
+  }
+}
+
+resource "aws_{service}_tag" "test" {
+  resource_arn = aws_{service}_{thing}.test.arn
+  key          = %[2]q
+  value        = %[3]q
+}
+`, rName, key, value)
+}
+```
+
+- Run `make testacc TEST=./aws TESTARGS='-run=TestAccAWS{Service}Tags_'` and ensure there are no failures.
+- In `website/aws.erb`: Add the new resource.
```

Comment:
This can be removed
```suggestion
```

Reply:
Updated in e87b2ea80831d4afa1359ba7762b5d0b7a3611f5 👍